### PR TITLE
Update code.cmd

### PR DESCRIPTION
**Pull request by longtsing**
Fixed the issue where the cmd window remains when executing the code command under cmd to open vscode.

The issue ：
https://github.com/microsoft/vscode/issues/210876
https://github.com/Microsoft/vscode/issues/70139

The reason for the bug is that when cmd opens vscode in the foreground, it will get stuck in the event loop of electron. 
So the cmd window can only exit automatically after vscode exits.
The solution is to open a new process and open vsocode in the background.

### CODE REVIEW

**Review by longtsing — COMMENTED** [review]
Add "START /B" to start a new process in the background to start vscode. The empty string is to fix the problem of the pop-up window prompting to select the js interpreter in the windows system.

**Review by longtsing — COMMENTED** [review]
It work in windows10

**Comment by longtsing** [comment]
@DonJayamanne May I have the Honor to contribute for the vscode?Please.

**Comment by DonJayamanne** [comment]
@longtsing The issue has been closed, please create a new issue so we know what you are trying to address.

**Comment by longtsing** [comment]
> @longtsing The issue has been closed, please create a new issue so we know what you are trying to address.
@DonJayamanne 

the new  issue  is these:
https://github.com/microsoft/vscode/issues/210876

**Comment by longtsing** [comment]
@DonJayamanne hello?

**Comment by DonJayamanne** [comment]
@longtsing I would suggest you first wait for the issue to get triaged by @roblourens 
Once the issue has been triaged, then the PR can be accepted.

File: resources/win32/bin/code.cmd
Comment (yunxu1019) [inline resources/win32/bin/code.cmd:1]: 我试了一下，windows10上确定能用，不过 /b 参数加不加都一样

Comment (longtsing) [reply resources/win32/bin/code.cmd:1]: 是的，不过加 /B 参数是为了保持兼容性，windows 10下可能加不加都一样，但是其他的版本的 windows 加个 /b 应该好一些。

Comment (longtsing) [reply resources/win32/bin/code.cmd:1]: 就等给 issue 分类，然后合并提交了。

**Comment by longtsing** [comment]
@roblourens

**Comment by roblourens** [comment]
See https://github.com/microsoft/vscode/issues/66750

**Pull request closed without merging**